Sources/TootSDK/Models/Suggestion.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Suggestion.swift
+//  
+//
+//  Created by Philip Chu on 5/17/23.
+//
+
+import Foundation
+
+/// An account suggested by the server.
+public struct Suggestion: Codable, Hashable, Sendable {
+    public init(source: Source, account: Account) {
+        self.source = source
+        self.account = account
+    }
+
+    /// The reason this account is being suggested.
+    public var source: Source
+    /// The account being recommended to follow.
+    public var account: Account
+
+    public enum Source: String, Codable, Sendable {
+        /// This account was manually recommended by your administration team
+        case staff
+        /// You have interacted with this account previously
+        case past_interactions
```

Comment:
```suggestion
        case pastInteractions = "past_interactions"
```

Reply:
Do we need this? (The decoder is set to handle camel case coming in)

Reply:
You're right, fixed my suggestion.

Reply:
@technicat just to say this is a great addition - if we can update this case to snake case, then I think this is good to go! 🙏

Reply:
Went with the first version of @Tunous suggestion, got a decoder error with the second. Here's how it looks now on universeodon.com

![IMG_2387](https://github.com/TootSDK/TootSDK/assets/1530082/588b47f3-5711-4d0f-b8d4-0e0c4cd2677f)

Reply:
Ah, camel case conversion works for fields not enum values. That's why my first suggestion worked :)